README.md

```diff
@@ -1,30 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> First project setting up a "Hello world" repository
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTLM
+- CSS
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
```

Comment:
- [ ]  Hi, your README is well done 👍 but since you dont' have a live link to your project you should remove this unnecessary lines of code